static/js/popovers.js

```diff
@@ -156,7 +156,7 @@ exports.toggle_actions_popover = function (element, id) {
             can_unmute_topic: can_unmute_topic,
             should_display_add_reaction_option: message.sent_by_me,
             should_display_edit_history_option: should_display_edit_history_option,
-            conversation_time_uri: narrow.by_conversation_and_time_uri(message),
+            conversation_time_uri: window.location .protocol + "//" + window.location.host + "/" + window.location.pathname.split('/')[1] + narrow.by_conversation_and_time_uri(message),
```

Comment:
the whitespace here is funny.  Also, I'd probably do this by adding a second option to that method that determines whether it should give an absolute URL, and calling, `narrow.by_conversation_and_time_uri(message, true)`